src/renderer/containers/Console/AddConnectionDialog.tsx

```diff
@@ -19,13 +19,15 @@ export interface AddConnectionDialogProps {
   selectedConnection: Partial<StoredConnection> | null;
   onSubmit: (conn: Omit<StoredConnection, "id">) => void;
   onCancel: () => void;
+  disableForm: boolean;
```

Comment:
Can we get this to just be disabled too?

```suggestion
  disabled?: boolean;
```